test/plugins/link.js

```diff
@@ -26,7 +29,7 @@ describe('Link plugin', function() {
     });
 
     this.irc.createMessage({
-      message: 'http://localhost:9002/basic'
+      message: 'http://127.0.0.1:9002/basic'
```

Comment:
@JocelynDelalande, I do think Travis CI cannot pass without that, as proven by https://github.com/Shuo-IRC/Shuo/pull/31, but I'm not entirely sure why.